frontend/src/utils/FileUtils.ts

```diff
@@ -1,9 +0,0 @@
-// eslint-disable-next-line import/prefer-default-export
```

Comment:
Can you double check this file is not used? Just thinking about CSV downloads.

Reply:
Searching for `downloadFile` in the code, I couldn't find any example of it. There's a whole `CSVUtils` file with the function `generateCSV` that handles CSV download.

Reply:
Great, thanks!